logs/build.go

```diff
@@ -21,6 +30,10 @@ func (cmd *buildCmd) Run(ctx context.Context, client *api.Client) error {
 		}, client.Project)
 	}
 
+	if build != nil {
+		cmd.Since = time.Since(build.CreationTimestamp.Time)
```

Comment:
Nit: can't this just be moved up into the `if cmd.Name != "" {` block after the `Get`? Then you don't need to nil check the `build`.

Reply:
yup 🤦